dev-docs/README.md

```diff
@@ -0,0 +1,40 @@
+# Developer Documentation
+
+This directory contains documentation for CatColab developers working on testing, deployment, and local development workflows.
+
+## Guides
+
+### [Testing Guide](docs/testing-guide.md)
+Learn how to run integration tests, test deployments, and debug test failures with a progressive isolation strategy.
+
+### [Development Workflows](docs/workflows.md)
+Example workflows demonstrating how to use `cc-utils` script for common development tasks.
```

Comment:
Running just `cc-utils`  on my system prints:

```
/home/kaspar/projects/topos/CatColab/infrastructure/scripts/cc-utils: line 29: $1: unbound variable
```

but exits with 0 i.e. success status while doing nothing. `cc-utils db` etc seems to be working.